File: SQLiteProject/app/src/main/java/com/tolgaalperkus/sqliteproject/MainActivity.java
package com.tolgaalperkus.sqliteproject;

import androidx.appcompat.app.AppCompatActivity;

import android.database.Cursor;
import android.database.SQLException;
import android.database.sqlite.SQLiteDatabase;
import android.os.Bundle;

import java.sql.SQLOutput;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        try{
            SQLiteDatabase database = this.openOrCreateDatabase("Musicians",MODE_PRIVATE,null);
            database.execSQL("CREATE TABLE IF NOT EXISTS musicians(id INTEGER PRIMARY KEY,name VARCHAR,age INT)");
            database.execSQL("INSERT INTO musicians(name,age) VALUES ('Lars',50)");
            database.execSQL("INSERT INTO musicians(name,age) VALUES ('James',60)");
            database.execSQL("INSERT INTO musicians(name,age) VALUES ('Kirk',55)");

            Cursor cursor = database.rawQuery("SELECT * FROM musicians",null);

            int idIx = cursor.getColumnIndex("id");
            int nameIx = cursor.getColumnIndex("name");
            int ageIx = cursor.getColumnIndex("age");

            while(cursor.moveToNext()){
                System.out.println("Name :"+cursor.getString(nameIx));
                System.out.println("Age :"+cursor.getInt(ageIx));
                System.out.println("Id :"+cursor.getInt(idIx));
            }
            cursor.close();

        } catch (SQLException e) {
            e.printStackTrace();
        }
    }
}

File: KotlinSQLite/app/src/main/java/com/tolgaalperkus/kotlinsqlite/MainActivity.kt
package com.tolgaalperkus.kotlinsqlite

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        try {

            val myDataBase = this.openOrCreateDatabase("Musicians", MODE_PRIVATE,null)
            myDataBase.execSQL("CREATE TABLE IF NOT EXISTS musicians (id INTEGER PRIMARY KEY,name VARCHAR, age INT)")
            myDataBase.execSQL("INSERT INTO musicians (name,age) VALUES ('James',50)")
            myDataBase.execSQL("INSERT INTO musicians (name,age) VALUES ('Lars',60)")
            myDataBase.execSQL("INSERT INTO musicians (name,age) VALUES ('Kirk',55)")

            myDataBase.execSQL("UPDATE musicians SET age = 61 WHERE name = 'Lars'")
            myDataBase.execSQL("UPDATE musicians SET name = 'Kirk Hammett' WHERE id = 3")
            myDataBase.execSQL("DELETE FROM musicians WHERE name = 'Lars'")

            val cursor = myDataBase.rawQuery("SELECT * FROM musicians ",null)
            //val cursorName = myDataBase.rawQuery("SELECT * FROM musicians WHERE name LIKE'%s'",null)
            val nameIx = cursor.getColumnIndex("name")
            val ageIx = cursor.getColumnIndex("age")
            val idIx = cursor.getColumnIndex("id")

            while (cursor.moveToNext()){
                println("Name : " + cursor.getString(nameIx))
                println("Age : "+ cursor.getInt(ageIx))
                println("Id : "+ cursor.getInt(idIx))
            }
            cursor.close()


        }catch (e : Exception){
            e.printStackTrace()
        }

    }
}